internal/cli/streams/privatelink/delete_test.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2025 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package privatelink
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/mocks"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDeleteOpts_Run(t *testing.T) {
```

Comment:
Does it make sense to have tests asserting the success and failure templates? I can see an argument that is validating the cli delete helper so feel free to tell me "no" 🙃

Reply:
I haven't found a clean way to check this in unit tests and we aren't verifying the output for other delete tests.

I added an additional test for the non-confirm delete case and we have an e2e test that verifies output for a successful delete.